service/synapse-service-rest/src/main/java/io/americanexpress/synapse/service/rest/service/BaseDeleteReactiveService.java

```diff
@@ -0,0 +1,14 @@
+package io.americanexpress.synapse.service.rest.service;
+
+import reactor.core.publisher.Mono;
+
+public abstract class BaseDeleteReactiveService extends BaseService {
```

Comment:
add javadocs to class and methods please
Also should naming be BaseReactiveDeleteService?